### PR TITLE
HAWKULAR-770 - keycloak.json via Servlet

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@pilhuhn , could you please review this one?

File: console/pom.xml [hunk 1]
```diff
@@ -55,9 +55,9 @@
       <version>2.0.12.Final</version>
     </dependency>
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>servlet-api</artifactId>
-      <version>2.5</version>
+      <groupId>org.jboss.spec.javax.servlet</groupId>
+      <artifactId>jboss-servlet-api_3.1_spec</artifactId>
```
Comment (pilhuhn) [inline console/pom.xml]: While not critical - is there no javax.servlet version of this?

**Comment by pilhuhn** [comment]
Thank you sir!

File: console/pom.xml (hunk 1 above)
Comment (jpkrohling) [inline console/pom.xml]: Not in the parent, from what I could check.

**Pull request merged**